packages/manager/core/generator/app/templates/src/components/GuideLink/guide-utils.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from 'react';
+import { Region } from '@ovh-ux/manager-config';
+import {
+  useAuthentication,
+  useEnvironment,
+} from '@ovh-ux/manager-react-core-application';
+
+const docUrl = 'https://docs.ovh.com/';
+
+type BaseUrlProps = {
+  [key in Region]: any;
+};
+
+const baseUrlPrefix: BaseUrlProps = {
+  EU: {
+    DEFAULT: `${docUrl}gb/en/`,
+    DE: `${docUrl}de`,
+    ES: `${docUrl}es`,
+    IE: `${docUrl}ie/en`,
+    IT: `${docUrl}it`,
+    PL: `${docUrl}pl`,
+    PT: `${docUrl}pt`,
+    FR: `${docUrl}fr`,
+    GB: `${docUrl}gb/en`,
+  },
+  CA: {
+    DEFAULT: `${docUrl}gb/en`,
+    CA: `${docUrl}ca/en`,
+    QC: `${docUrl}ca/fr`,
+    WE: `${docUrl}us/en`,
+    WS: `${docUrl}us/en`,
+  },
+  US: {
+    DEFAULT: `${docUrl}gb/en`,
+    US: `${docUrl}us/en`,
+  },
+};
+
+const GuidesList: any = {
```

Comment:
```suggestion
type GuideLinks = {
  DE: string;
  ES: string;
  IE: string; 
  IT: string;
  PL: string;
  PT: string;
  FR: string;
  GB: string;
  CA: string;
  QC: string;
  WE: string;
  WS: string;
  US: string;
};

const guidesList: { [guideName: string]: Partial<GuideLinks> }  = {
```

`guidesList` or `GUIDES_LIST` since this is a constant